sys-fs/copyfs/copyfs-1.0.1-r1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="fuse-based filesystem for maintaining configuration files"
+HOMEPAGE="https://boklm.eu/copyfs/"
+SRC_URI="https://boklm.eu/copyfs/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~x86 ~amd64 ~ppc"
+
+DEPEND="sys-apps/attr
+	>=sys-fs/fuse-2.0"
```

Comment:
what about `RDEPEND`?

Reply:
AFAICS, fuse should be in RDEPEND (as it's needed to work with the program), attr in DEPEND. Updated the ebuild.